doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -898,11 +898,11 @@ In principle, computing extraction efficiency first involves computing the radia
 
 To compute the radiation pattern $P(\theta, \phi)$ requires three steps:
 
-1. For each simulation in the Fourier-series expansion ($m = 0, 1, ..., M$), compute the far fields $\vec{E}_m$, $\vec{H}_m$ for the desired $\theta$ points in the $rz$ ($\phi = 0$) plane, at an "infinite" radius (i.e., $R \gg \lambda$) using a [near-to-far field transformation](../Python_User_Interface.md#near-to-far-field-spectra).
-2. Obtain the *total* far fields at these points, for a given $\phi$ by summing the far fields from (1): $\vec{E}_{tot}(\theta, \phi) = \vec{E}_{m=0}(\theta)e^{im\phi} + 2\sum_{m=1}^M \Re\{\vec{E}_m(\theta)e^{im\phi}\}$ and $\vec{H}_{tot}(\theta, \phi) = \vec{H}_{m=0}(\theta)e^{im\phi} + 2\sum_{m=1}^M \Re\{\vec{H}_m(\theta)e^{im\phi}\}$.  Note that $\vec{E}_m$ and $\vec{H}_m$ are generally complex, and are conjugates for $\pm m$.
+1. For each simulation in the Fourier-series expansion ($m = -M, \ldots , 0, \ldots, M$), compute the far fields $\vec{E}_m$, $\vec{H}_m$ for the desired $\theta$ points in the $rz$ ($\phi = 0$) plane, at an "infinite" radius (i.e., $R \gg \lambda$) using a [near-to-far field transformation](../Python_User_Interface.md#near-to-far-field-spectra).
+2. Obtain the *total* far fields at these points, for a given $\phi$ by summing the far fields from (1): $\vec{E}_{tot}(\theta, \phi) = \vec{E}_{m=0}(\theta)e^{im\phi} +\sum_{m=-M}^M \vec{E}_m(\theta)e^{im\phi}$ and $\vec{H}_{tot}(\theta, \phi) = \vec{H}_{m=0}(\theta)e^{im\phi} + \sum_{m=-M}^M \vec{H}_m(\theta)e^{im\phi}$.  Note that $\vec{E}_m$ and $\vec{H}_m$ are generally complex.  (The $\pm m$ terms are related by a mirror flip in $\phi$, and also by complex conjugation if you also flip the sign of the frequency, so it is possible to combine their calculation.)
```

Comment:
To improve clarity, consider changing the phase "..., and also by complex conjugation..." to "..., and if you form the full expression for the fields by adding the phase term $e^{im\phi-i\omega t}$ are related  by complex conjugation if you also flip the sign of the frequency $\omega$".

Reply:
(Note that we have already verified that the $\pm m$ terms are related by a mirror flip in $\phi$. See [#2950 (comment)](https://github.com/NanoComp/meep/pull/2950#issuecomment-2564145876).)

Reply:
```suggestion
2. Obtain the *total* far fields at these points, for a given $\phi$ by summing the far fields from (1): $\vec{E}_{tot}(\theta, \phi) = \vec{E}_{m=0}(\theta)e^{im\phi} +\sum_{m=-M}^M \vec{E}_m(\theta)e^{im\phi}$ and $\vec{H}_{tot}(\theta, \phi) = \vec{H}_{m=0}(\theta)e^{im\phi} + \sum_{m=-M}^M \vec{H}_m(\theta)e^{im\phi}$.  Note that $\vec{E}_m$ and $\vec{H}_m$ are generally complex.  (The $\pm m$ terms are related by a mirror flip in $\phi$, and also by complex conjugation if you also flip the sign of the frequency and conjugate the source, so it is possible to combine their calculation.)
```